mayastor/src/bdev/nexus/nexus_io.rs

```diff
@@ -87,6 +87,13 @@ impl Bio {
         unsafe { Bdev::from((*self.0).bdev) }
     }
 
+    pub fn new(pio: *mut spdk_bdev_io, in_flight: i8) -> Bio {
```

Comment:
Should have a comment for each function.

Reply:
Ok, I'll fix that.